apps/web/pages/[username]/[collectionId]/[tokenId]/index.tsx

```diff
@@ -43,6 +43,20 @@ export default function NftDetailPage({ username, collectionId, tokenId }: NftDe
     return <GalleryRedirect to={{ pathname: '/' }} />;
   }
 
+  // Check if the url params match the legacy url format for Artblocks and Prohibition collection pages and if so redirect to the new collection page url.
+  // We updated the Artblocks and Prohibition collection page urls in Feb 2024 to add "/community/".
+  // Previously they were formatted like "/artblocks/0x123/123" which fits the nft detail page url pattern
+  if ((username === 'artblocks' || username === 'prohibition') && collectionId.startsWith('0x')) {
+    return (
+      <GalleryRedirect
+        to={{
+          pathname: `/community/${username}/[contractAddress]/[projectId]`,
+          query: { contractAddress: collectionId, projectId: tokenId },
+        }}
+      />
+    );
+  }
```

Comment:
nit: this should be configurable in next.config.js ([our file](https://github.com/gallery-so/gallery/blob/main/apps/web/next.config.js#L88)), but you'll need to do some regex trickery

tbh i don't think it's worth spending the time on given we probably won't even hit this redirect very often